charts/dftbackend/templates/deployment.yaml

```diff
@@ -1,6 +1,5 @@
-#################################################################################
-# Copyright (c) 2021,2022 Catena-X
-# Copyright (c) 2021,2022 Contributors to the Eclipse Foundation
+###############################################################
+# Copyright (c) 2021, 2023 Contributors to the Eclipse Foundation
```

Comment:
please add 2022 in year list in copyright header

Reply:
Copy header added.